doc: fix powershell redir

janlazo opened the pull request:
Tested using with my vimrc, https://github.com/junegunn/vim-plug/pull/860, on Vim and Neovim.

janlazo committed:
doc: fix powershell redir

justinmk commented on runtime/doc/options.txt:
```diff
@@ -5159,8 +5159,9 @@ A jump table for the options with a short description can be found at |Q_op|.
-		set shell=powershell shellquote=( shellpipe=\| shellredir=> shellxquote=
+		set shell=powershell shellquote=( shellpipe=\| shellxquote=
 		set shellcmdflag=-NoLogo\ -NoProfile\ -ExecutionPolicy\ RemoteSigned\ -Command
+		set shellredir=\|\ Out-File\ -Encoding\ ASCII
```
would this be a problem for utf-8?

janlazo replied:
https://www.w3.org/2001/06/utf-8-test/UTF-8-demo.html
`ASCII` encoding corrupts the file.

```ps1
Get-Content -Encoding UTF8 UTF-8-demo.html | Out-File foo.txt
```
foo.txt seems to retain the utf-8 characters if omitting `-Encoding` or using `utf8`, `utf32`, `unicode` encoding values.

justinmk commented on runtime/doc/options.txt:
```diff
@@ -5159,8 +5159,9 @@ A jump table for the options with a short description can be found at |Q_op|.
 							*shell-powershell*
 	To use powershell (on Windows): >
-		set shell=powershell shellquote=( shellpipe=\| shellredir=> shellxquote=
+		set shell=powershell shellquote=( shellpipe=\| shellxquote=
```
I wonder what `>`  is equivalent to exactly.

janlazo replied:
utf-16 on Windows.
utf-8 on Unix.

janlazo committed:
doc: use utf8 encoding to be safe

Encoding can be utf8, unicode, utf32.
User can choose to omit '-Encoding' to default to 'unicode'.

'ascii' encoding corrupts the following file:
https://www.w3.org/2001/06/utf-8-test/UTF-8-demo.html

Inspect 'foo.txt' with the following code after downloading the file.

Get-Content -Encoding UTF8 UTF-8-demo.html | Out-File -Encoding ascii foo.txt

blueyed commented:
Should this be done here then also (at least)?
https://github.com/blueyed/neovim/blob/374a35b044ce324076bd7639dd4db08952f245b3/test/functional/helpers.lua#L498-L503

janlazo commented:
Sure. Neovim can handle `set shellredir=>` but my shellredir suggestion should work for the functional tests.
